CityPuzzle/Classes/Sql.cs

```diff
@@ -63,7 +63,21 @@ public static String ReadUsers()
             }
         }
 
-
+        public static void SavePuzzle(Puzzle puzzle)
+        {
+            using (SqlConnection conn = new SqlConnection(ConnStr))
+            {
+                conn.Open();
+                var command = new SqlCommand("INSERT INTO Puzzles (Name,About,Quest,Latitude,Longitude) VALUES (@Name,@About,@Quest,@Latitude,@Longitude)", conn);
```

Comment:
Could use Entity Framework to save whole object in one command

Reply:
I'll fix it later. Now I want to end with a simple idea of sql input.